src/topic.rs

```diff
@@ -4,6 +4,8 @@
 
 use uuid::Uuid;
 
+pub use openfmb_messages::{Module, Profile};
```

Comment:
Should I leave this public, allows topic::Module and topic::Profile to be used, I could remove the Module and Profile from the top level crate then and force users of the crate to prefix with topic::Module and topic::Profile rather than simply Module and Profile (maybe easier to read?)